src/v/storage/tests/storage_e2e_test.cc

```diff
@@ -1027,6 +1028,14 @@ FIXTURE_TEST(write_concurrently_with_gc, storage_test_fixture) {
  */
 FIXTURE_TEST(append_concurrent_with_prefix_truncate, storage_test_fixture) {
     auto cfg = default_log_config(test_dir);
+    // start stress fiber to make the test less likely to trigger race
+    // conditions
+    stress_fiber_manager stress_mgr;
+    stress_config stress_cfg;
+    stress_cfg.min_spins_per_scheduling_point = 100;
```

Comment:
does it make sense to repeat this test with slightly different parameters (maybe even randomized)?

Reply:
good point, i can do it, those params reproduced the bug in 100% of cases